cmd/cli/kubectl-kyverno/processor/result.go

```diff
@@ -144,3 +144,19 @@ func (rc *ResultCounts) AddValidatingPolicyResponse(engineResponse engineapi.Eng
 		}
 	}
 }
+func (rc *ResultCounts) AddMutatingAdmissionPolicyResponse(engineResponse engineapi.EngineResponse) {
+	for _, ruleResp := range engineResponse.PolicyResponse.Rules {
+		switch ruleResp.Status() {
+		case engineapi.RuleStatusPass:
+			rc.Pass++
+		case engineapi.RuleStatusFail:
+			rc.Fail++
+		case engineapi.RuleStatusError:
+			rc.Error++
+		case engineapi.RuleStatusWarn:
+			rc.Warn++
+		case engineapi.RuleStatusSkip:
+			rc.Skip++
+		}
+	}
+}
```

Comment:
It is no longer needed, as you used the existing function `processMutateEngineResponse` so we can remove it.